main.browser.js

```diff
@@ -0,0 +1,7 @@
+require('babel-polyfill');
+const Client4 = require('./client/client4.js').default;
```

Comment:
Why do this instead of using webpack's library target? Then we can import using import {} from